### DISCUSSION TITLE
Github has a bug meaninglessly charges $200 for each transaction even it is free

### DISCUSSION THREAD

**Discussion by yusuf-mutlu** [community/community#30542] [Copilot Conversations]
Hi friends,

Github started to feel like a scam project which we know it isn't.  Yesterday(22/08/2022) I first subscribed to Github Copilot because it was expiring . After I subscribed to copilot, I wanted to cancel my subscription because I didn't want it to renew next year.
After I've done above steps I saw from my card I was charged $300. I submitted a support ticket yesterday for it and I still have no response.

Another charge of $200 happened today(23/08/2022). I just wanted to try free CircleCI application inside github. It says it is free ($0/Month). I subscribed to it because it was free and I was charged another $200 more. 

In total I was charged $500 and I was just trying to renew my copilot subscription. They still have this bug, please be careful.

![image](https://user-images.githubusercontent.com/4123648/186180325-02437f46-98d9-4ec7-9004-9a9077b7b773.png)

**Comment by lcremona** [comment]
Same here. I opened a ticket yesterday and still no response

**Comment by yusuf-mutlu** [comment]
After I limited my monthly spending to $1, Github charged me $100 more. 

That's like a bitter joke. What's happening? They charged me $500 more for nothing. I was charged $600 in total!!

![image](https://user-images.githubusercontent.com/4123648/186182826-70b98d65-9a57-4839-a387-fc25d2f3e642.png)

**Comment by yusuf-mutlu** [comment]
Like a joke 
![image](https://user-images.githubusercontent.com/4123648/186184074-fa2e689d-ea8e-4eb2-8fac-8b0c4599a275.png)

**Reply by strugglebak** [reply]
I think you need to unbundle your bank card, etc.

**Comment by lcremona** [comment]
The temporary solution I found to not be charge again is to remove the cards from my paypal and change it on github so that it charges me from paypal. This does not work if you also have a paypal balance.

**Reply by yusuf-mutlu** [reply]
I just froze my card.

**Comment by iamcjwilliams** [comment]
Hi there! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them and they should be able to help you get this sorted :)

**Reply by yusuf-mutlu** [reply]
I told in my comment that I submitted a support ticket yesterday. They still didn't answer it. They ignore the issue and they make no explanation about it. This is really a serious bug.

**Reply by iamcjwilliams** [reply]
We're sorry for the trouble here. Our GitHub Support team will reach out as soon as possible. Please note that due to current high volume, there may be a longer than usual wait time :)

**Reply by josejalvarezm** [reply]
Thank you for your comment, please could you let us know how many days it can take? An explanation of this glitch is much appreciated.

**Reply by josejalvarezm** [reply]
> We're sorry for the trouble here. Our GitHub Support team will reach out as soon as possible. Please note that due to current high volume, there may be a longer than usual wait time :)

hm... Am I speaking to a GITHUB bot?

**Comment by josejalvarezm** [comment]
Same here, they charged me 3 times for github copilot. I raised two tickets and still waiting for their response.

What's going on? GITHUB is supposed to deliver a robust subscription management!

I am sorry to say it is not rocket science.

Regards.

**Reply by josejalvarezm** [reply]
Many thanks I received the refunds. Kind regards.

**Comment by yusuf-mutlu** [comment]
They replied support ticket and refunded $500 as I wanted.

**Comment by lcremona** [comment]
Good! me too

**Comment by WalidEl3mary** [comment]
Same here. I submitted a ticket yesterday but no response till now.

**Reply by josejalvarezm** [reply]
It happened to me and it took around 1 day to have the refund, hopefully you will have yours soon. Regards.
